static/app/components/dynamicSampling/investigationRule.tsx

```diff
@@ -174,34 +178,20 @@ const InvestigationInProgressNotification = styled('span')`
   gap: ${space(0.5)};
 `;
 
-function checkIsTransactionQueryMissing(error: RequestError | null) {
```

Comment:
since the request won't be fired if `event.type='transaction'` is not present in the query, this won't happen anymore - therefore I am removing this code